app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="form-container">
+<br>
+<h2>Log in</h2>
+
+<%= bootstrap_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Nice job making the signup/login page work 👌 However, this activity requires the modification of the views in _devise_ for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly for both **Login** and **Signup** pages, as seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your Signup view:**        |  **Your login view:**        |  **Wireframe login view (Required):**
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/matee-safi/Blog-app/assets/12745474/a399c01e-c711-4eb3-908b-b6136e121224"> | <img width="800" alt="Home Page" src="https://github.com/matee-safi/Blog-app/assets/12745474/11035f1f-8912-4413-89cb-8619c65f5f8f"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png">